pkg/ebpf/c/types.h

```diff
@@ -1,10 +1,11 @@
-#ifndef __TRACEE_TYPES_H__
-#define __TRACEE_TYPES_H__
+#ifndef __TYPES_H__
+#define __TYPES_H__
 
 #include <vmlinux.h>
-#include <missing_definitions.h>
+#include <vmlinux_flavors.h>
```

Comment:
I don't see a reason to have this header here

Reply:
ditto (tried to create a pattern, can remove it).